webapp/src/main/java/org/apache/atlas/web/filters/AtlasXSSPreventionFilter.java

```diff
@@ -0,0 +1,208 @@
+    private static final Pattern REGEX_NUMBER                   = Pattern.compile("^[-+]?[0-9]*\\.?[0-9]+([eE][-+]?[0-9]+)?$");
+    public  static final Pattern REGEX_INTEGER                  = Pattern.compile("^[0-9]+$");
+    public static final  Pattern REGEX_ISO8601                  = Pattern.compile("^^[0-9]{4}(-[0-9]{2}(-[0-9]{2}([ T][0-9]{2}(:[0-9]{2}){1,2}(.[0-9]{1,6})"
+            +"?Z?([\\+-][0-9]{2}:[0-9]{2})?)?)?)?$");
+    public static final  Pattern REGEX_PARAGRAPH                = Pattern.compile("^[\\p{L}\\p{N}\\s\\-_',\\[\\]!\\./\\\\\\(\\)]*$");
+    public static final  Pattern REGEX_SPACE_SEPARATED_TOKENS   = Pattern.compile("^([\\s\\p{L}\\p{N}_-]+)$");
+    public static final  Pattern REGEX_DIRECTION                = Pattern.compile("(?i)^(rtl|ltr)$");
+    public static final  Pattern REGEX_IMAGE_ALIGNMENT          = Pattern.compile("(?i)^(left|right|top|texttop|middle|absmiddle|baseline|bottom|absbottom)$");
+    public static final  Pattern REGEX_NUMBER_OR_PERCENT        = Pattern.compile("^[0-9]+[%]?$");
+    public static final  Pattern REGEX_LIST_TYPE                = Pattern.compile("(?i)^(circle|disc|square|a|A|i|I|1)$");
+    public static final  Pattern REGEX_CELL_ALIGN               = Pattern.compile("(?i)^(center|justify|left|right|char)$");
+    public static final  Pattern REGEX_CELL_VERTICAL_ALIGN      = Pattern.compile("(?i)^(top|middle|bottom|baseline)$");
+    public static final  Pattern REGEX_SHAPE                    = Pattern.compile("(?i)^(rect|circle|poly|default)$");
+    public static final  Pattern REGEX_LANG                     = Pattern.compile("^[a-zA-Z]{2,20}");
+    public static final  Pattern REGEX_ID                       = Pattern.compile("[a-zA-Z0-9\\:\\-_\\.]+");
+    public static final  Pattern REGEX_NAME                     = Pattern.compile("^([\\p{L}\\p{N}_-]+)$");
+    public static final  Pattern REGEX_USEMAP                   = Pattern.compile("(?i)^#[\\p{L}\\p{N}_-]+$");
+    public static final  Pattern REGEX_OPEN                     = Pattern.compile("(?i)^(|open)$");
+    public static final  Pattern REGEX_COORDS                   = Pattern.compile("^([0-9]+,)+[0-9]+$");
+    public static final  Pattern REGEX_SCOPE                    = Pattern.compile("(?i)(?:row|col)(?:group)?");
+    public static final  Pattern REGEX_NOWRAP                   = Pattern.compile("(?i)|nowrap");
+    public static final  Pattern REGEX_ONSITE_URL               = Pattern.compile("(?:[\\p{L}\\p{N}\\\\\\.\\#@\\$%\\+&;\\-_~,\\?=/!]+|\\#(\\w)+)");
+    public static final  Pattern REGEX_OFFSITE_URL              = Pattern.compile("\\s*(?:(?:ht|f)tps?://|mailto:)[\\p{L}\\p{N}]"
+            + "[\\p{L}\\p{N}\\p{Zs}\\.\\#@\\$%\\+&;:\\-_~,\\?=/!\\(\\)]*+\\s*");
+    public static final  Predicate<String> REGEX_ON_OFFSITE_URL = matchesEither(REGEX_ONSITE_URL, REGEX_OFFSITE_URL);
+
+    @Inject
```

Comment:
Do we need an inject here?

Reply:
No constructor parameter so we can ommit it.